examples/docs_snippets/docs_snippets/concepts/assets/graph_backed_asset.py

```diff
@@ -84,7 +84,7 @@ def middle_asset(upstream_asset):
     return add_one(upstream_asset)
 
 
-middle_asset = AssetsDefinition.from_graph(middle_asset)
+middle_asset = AssetsDefinition.from_graph(middle_asset)  # type: ignore[assignment]
```

Comment:
Not your fault, but we shouldn't be doing this kind of reassignment in a user-facing example. I think it would be best to give the graph a different name like `middle_asset_graph`.